README.md

```diff
@@ -36,17 +36,43 @@ or warning it is not installed, also it will create the gemset.
 rvm install ruby-2.4.0
 ```
 
-##### Database creation
+#### Database setup.
 ```
-rais db:setup
+rails db:setup
 ```
 
-##### Database initialization
+#### Install bundler gem at global gemset.
+
+```
+rvm @global do gem install bundler
+```
+
+#### Running falae
+
+Installing gems:
+
 ```
-rails db:seed
+bundle install
 ```
+Set FALAE_IMAGES_PATH environment variable, which should points to a directory where pictograms and users images will be stored. It should not set this variable to the app public directory.
```

Comment:
I'd change this paragraph to: `Set FALAE_IMAGES_PATH environment variable to point to a directory where pictograms and user's images will be stored in. This variable should not point to the app's public directory.`